101-backup-vm-using-new-recoveryservices-vault/azuredeploy.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vaultName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Vault"
+      }
+    },
+    "registeredContainers": {
+      "type": "array",
+      "metadata": {
+        "description": "Array of registered containers. It will take input as type of container; vm name; cloud service name. e.g for iaasvm type: iaasvmcontainer;myvm1;myvm1"
+      }
+    }
+
+  },
+  "resources": [
+    {
+      "type": "Microsoft.RecoveryServices/vaults",
+      "apiVersion": "2015-11-10",
+      "name": "[parameters('vaultName')]",
+      "location": "[resourceGroup().location]",
+      "sku": {
+        "name": "RS0",
+        "tier": "Standard"
+      },
+      "properties": {
+      }
+    }, 
+    {
+
+            "name": "[concat(parameters('vaultName'), '/', variables('backupFabric'), '/', parameters('registeredContainers')[copyIndex()], '/', parameters('existingProtectableItems')[copyIndex()])]",
+
+            "apiVersion": "2015-11-10",
+
+            "location": "[resourceGroup().location]",
+            "dependsOn": [ "[concat('Microsoft.RecoveryServices/vaults/', parameters('vaultName'))]" ],
+
+            "type": "Microsoft.RecoveryServices/vaults/backupFabrics/protectionContainers/protectedItems",
+
+            "copy": {
+
+                "name": "protectedItemsCopy",
+
+                "count": "[length(parameters('registeredContainers'))]"
+
+            },
+
+            "properties": {
+
+                "protectedItemType": "[variables('protectedItemType')]",
+
+                "policyName": "DefaultPolicy"
```

Comment:
Add sourceResourceId as a property. And if you update API version to 2016-06-01 then pass policyId instead of policyName.